container/openejb-loader/src/main/java/org/apache/openejb/observer/ObserverManager.java

```diff
@@ -74,6 +80,12 @@ public boolean addObserver(final Object observer) {
         }
     }
 
+    /**
+     * Remove the observable object so this object doesn't have any observers
```

Comment:
I think only specified `observer` will be removed, and all the rest will be kept.
And observable will not be removed.